{
  "cveId": "CVE-2019-25165",
  "eventName": "New CVE Received",
  "cveChangeId": "F61AB5DB-4F97-47F8-A70E-D7DD2B2B1039",
  "sourceIdentifier": "416baaa9-dc9f-4396-8d5f-8c081fb06d67",
  "created": "2025-09-16T16:15:34.103",
  "details": [
    {
      "action": "Added",
      "type": "Description",
      "newValue": "Rejected reason: ** REJECT ** DO NOT USE THIS CVE RECORD. ConsultIDs: none. Reason: This record was in a CNA pool that was not assigned to any issues during 2019. Notes: none."
    }
  ]
}